libs/sdk-ui-kit/src/AppearancePage/WhiteLabeling.tsx

```diff
@@ -0,0 +1,73 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { Button } from "../Button";
+import { TooltipIcon } from "./TooltipIcon";
+import { DocumentationLink } from "./DocumentationLink";
+import { LoadingSpinner } from "../LoadingSpinner";
+import { StylingComponent } from "./StylingComponent";
+import { StylingHeader } from "./StylingHeader";
+import { StylingFooter } from "./StylingFooter";
+
+const ALIGN_POINTS = [{ align: "cr tl" }, { align: "bc tc" }, { align: "bc tr" }];
+const ARROW_OFFSETS = { "cr tl": [16, -12], "bc tc": [0, 16], "bc tr": [12, 16] };
+
+/**
+ * @internal
+ */
+export interface IWhiteLabelingProps {
+    title: string;
+    status: string;
+    tooltip: string;
```

Comment:
Maybe `titleTooltip` would be better so it is obvious where the tooltip will be used if you don't know how the component looks like.